### ISSUE TITLE
Distribute as Flatpak

### ISSUE THREAD

**Issue by gianni-rosato** [Alex313031/Mercury#10]
[LibreWolf](https://beta.flathub.org/apps/io.gitlab.librewolf-community) is available as a Flatpak, as is standard [Mozilla Firefox](https://beta.flathub.org/apps/org.mozilla.firefox).

Scrolling down & looking at the number of downloads these browsers receive, LibreWolf gets consistently over 650 downloads _per day_ & Firefox gets over 7,000 per day. Mercury is incredible, and I think the average user would enjoy its benefits. Purely from an adoption standpoint, I could see Flatpak distribution doing great things for Mercury. Many distros are configured with Flatpak support out of the box now, and it would make Mercury far more accessible to the average user. I really hope you consider this, as I'd love to use a Mercury Flatpak & I'm sure many others would as well. Thank you!

**Comment by gregorni** [comment]
I agree, especially since Flathub is becoming the most popular option for both users and developers.

**Comment by emanuelserpa** [comment]
https://gitlab.com/librewolf-community/browser/flatpak
https://github.com/flathub/io.gitlab.librewolf-community

LibreWolf can be useful as a reference.

**Comment by xplshn** [comment]
If you write the YAML, I could mantain the package as you update this project, I already mantain a FlatPak package, OpenARENA(Quake3e engine).

**Comment by gianni-rosato** [comment]
Hey, any word on whether or not this is being considered?

**Comment by xplshn** [comment]
Hasn't said anything yet :(

**Comment by Lenny5125** [comment]
Would love to see this! Flatpak would make installing cleaner and easier.

**Comment by gz83** [comment]
thank you for your suggestions

**Comment by nhubaotruong** [comment]
Hi @gz83 @xplshn I've created flathub manifest based on librewolf, can you help submit it to flathub and maintain it ?
https://github.com/nhubaotruong/mercury-flatpak

**Comment by gz83** [comment]
> Hi @gz83 @xplshn I've created flathub manifest based on librewolf, can you help submit it to flathub and maintain it ? https://github.com/nhubaotruong/mercury-flatpak

Have you tested it?

@nhubaotruong

**Comment by nhubaotruong** [comment]
Yes, I've built the flatpak and can run it locally, I just hasn't got the icon right yet
![image](https://github.com/Alex313031/Mercury/assets/28810481/3c0413d8-e52a-4203-bf8b-1f1f5c1d12b3)

**Comment by gz83** [comment]
Which icon is abnormal?

@nhubaotruong

**Comment by nhubaotruong** [comment]
@gz83 I meant I haven't put in any app icon and .desktop file

**Comment by xplshn** [comment]
I'm too occupied with school, I will check your work tho!

**Comment by rguards** [comment]
I created a submission in [there](https://github.com/flathub/flathub/pull/5308)
